Code/lib/motor/motor.h

```diff
@@ -34,7 +34,8 @@ class Drive
     int turn_delay = 1000;
     // The speed of motor during turning
     int turn_speed = 125;
-
+    //Default drive speed
+    int drive_speed = 125;
```

Comment:
this value should be configurable from initialization of the class or through a set method

Reply:
On it